core/templates/dev/head/pages/splash.html

```diff
@@ -8,30 +8,114 @@
+      <img src="images/splash/Puzzles.svg" class="oppia-splash-back-images" />
+
+      <img src="images/splash/Humor.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Philosophy.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Mathematics.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Religion.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Cooking.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Engineering.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Microbiology.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/History.svg" class="oppia-splash-back-images" />
+      <img src="images/splash/Architecture.svg" class="oppia-splash-back-images" />
+    </div>
+    <img src="images/splash/books.svg" id="oppia-splash-books" />
+    <h2 class="oppia-splash-h2">Oppia makes it easy to create interactive explorations<br>that educate and engage.</h2>
+    <div class="row oppia-splash-buttons-row">
+      <button class="oppia-splash-buttons">BROWSE OUR EXPLORATIONS</button>
+      <button class="oppia-splash-buttons">CREATE YOUR OWN EXPLORATIONS</button>
     </div>
   </div>
 
-  {% include 'components/profile_link_image_directive.html' %}
-  {% include 'components/circular_image_directive.html' %}
-{% endblock %}
+  <div id="oppia-splash-section-two">
+    <div class="oppia-splash-text">
+      <div class="row oppia-splash-bullet-row">
+        <div class="col-md-6">
+          Explorations are interactive lessons<br>that provide more immersive<br>experiences than static videos or text,<br>helping users learn by doing.
+        </div>
+        <div class="col-md-6">
+          <img class="oppia-splash-images" src="images/splash/bullet1icon.svg" />
+        </div>
+      </div>
+
+      <div class="row oppia-splash-bullet-row">
+        <div class="col-md-6">
+          <img class="oppia-splash-images" src="images/splash/bullet2icon.svg" />
+        </div>
+        <div class="col-md-6">
+          Explorations are simple to create. They are<br>easily adapted based on individual student<br>feedback and trends in the experience of<br>your learners around the world.
+        </div>
+      </div>
+
+      <div class="row oppia-splash-bullet-row">
+        <div class="col-md-6">
+          Oppia lets you create explorations<br>on a wide range of subjects,<br>limited only by your imagination.
+        </div>
+        <div class="col-md-6">
+          <img class="oppia-splash-images" src="images/splash/bullet3icon.svg" />
+        </div>
+      </div>
+    </div>
+
+  </div>
 
-{% block footer_js %}
-  {{ super() }}
   <script>
-    {{ include_js_file('components/ProfileLinkImageDirective.js') }}
-    {{ include_js_file('components/CircularImageDirective.js') }}
+    // This scrolls the page to the anchor clicked on in the quick links menu.
```

Comment:
I'm not sure about this, I took the general structure from another page. Please let me know what is needed to be done here.